controllers/games.controllers.js

```diff
@@ -75,11 +76,28 @@ const updateVotes = (req, res, next) => {
     .catch((err) => next(err));
 };
 
+const deleteComment = (req, res, next) => {
+  const commentId = req.params.comment_id;
+
+  let parsedCommentId;
+  if (!isNaN(commentId)) {
```

Comment:
We might not need to manually check for a valid comment_id here. What happens if we run this invalid id through the sql query? We may be able to just catch the error that psql provides.